discovery/src/main/java/com/ibm/watson/discovery/v2/query/GenericQueryAggregation.java

```diff
@@ -1,12 +0,0 @@
-package com.ibm.watson.discovery.v2.query;
```

Comment:
Same with this file

Reply:
This is a custom file that was deprecated in previous releases for handling aggregations. So given the way we handle discriminators now in the SDK and because it's a major release I felt safe removing it.